### DISCUSSION TITLE
Add shortcuts to display double pages

### DISCUSSION THREAD

**Discussion by SavanFlou** [gotson/komga#1777] [Ideas]
Hello,
it is my first app I am using to read my comics and what a change!
But I usually use an old tablet to read them, and from time to time, I would need to see in double page mode, but display the menu, hit option, choose one in the list,, see it... then do it all over again to display just one page... it is a bit long. Even more when I need it every 4 pages 😄 
So one option could be to add a shortcut (or maybe shortcuts) for this option,  next to the fullscreen button for example, in order to have a direct access to this feature. And for people (inccluding me from time to time) reading on real computers, maybe adding keyboard shortcuts?

**Comment by gotson** [comment]
Why would you need to change it so often? I don't get it.

**Comment by SavanFlou** [comment]
Because from time to time, a drawing is displayed on 2 pages, but the rest of the comic is "normal", you can read it with just one page visible. 
So I need to look at the page to see the texts, but I would like to be able to display the 2 pages quickly to enjoy the drawings. And put back the "normal" display for the rest of the comic. 
And with some, you have a 2-pages drawing, then one page, then 2, then one, etc.

**Reply by gotson** [reply]
there's a shortcut but using keyboard, it's not practical to support shortcuts for mobile UI.

**Comment by SavanFlou** [comment]
True, I did not see that at first but "d" allows us to do what I am talking about.
And true also, on a mobile, or on my tablet, keyboard shortcuts are not very pratical. This is why I was thinking about adding one shortcut to the Webreader's interface.

**Comment by radonmiser** [comment]
When I brought up the same issue last time, I was suggested to use 3rd party apps instead for a better UX on touchscreen devices. But alas none of them are as good as the default web reader imho, so I hacked together a userscript to save myself from constantly going deep in the menu for a function that's so frequently used. I hope this is implemented in the web UI because apple users deserve the convenience too.
https://gist.github.com/radonmiser/f79a7c4c56d5c5e379c8318bd3957e6b

**Reply by radonmiser** [reply]
I have so far found the reading mode shortcuts in the 3-finger tap menu to be pretty useless, series metadata already covered that. You can modify it to trigger more useful keys.

**Comment by SavanFlou** [comment]
But metadatas mean you have to be aware of it. So either you correct the problem after reading the comics (and you note the pages) or you must be able to do it directly, complicated on my small tablet or a mobile phone, or you must first look at the pages and see if there are some that are connected and edit the metadatas to do so.
I do not see that as "easy".
More useful keys, ok why not but which ones?
This app is meant to read comics, this is a quite current pattern to have a big drawing on multiple pages. I think being able to quickly see pages by 2 or by 1, by just one action, is a feature REALLY useful.

Using 3rd parties readers... Why? If so, this feature is considered as important if others put it. And so, why not add it directly in the app?
Maybe there are problems to add it or others, I can understand this reason, but besides that, why not trying to improve the internal reader?

**Reply by radonmiser** [reply]
Perhaps I didn't express myself well, I fully agree with you that's why I made the userscript to allow you to use two-finger tap to quickly switch between double page and single page.
The 3-finger tap menu that came with the above script I found not so useful, so I suggested people to repurpose it as they see fit.

**Comment by SavanFlou** [comment]
My bad, I read too fast and saw "3 digits" instead of "3 fingers". So for me you were talking about the menu in the reader.
Will look into your script, could  be a solution.
